src/providers/dart_commands.ts

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+import * as as from "../analysis/analysis_server_types";
+import * as editors from "../editors";
+import * as vs from "vscode";
+import { Analyzer } from "../analysis/analyzer";
+
+export class DartCommands  {
+	private analyzer: Analyzer;
+	constructor(context: vs.ExtensionContext, analyzer: Analyzer) {
+		this.analyzer = analyzer;
+
+        context.subscriptions.push(
+            vs.commands.registerTextEditorCommand("dart.organizeDirectives", this.organizeDirectives, this)
+        );
+	}
+
+    organizeDirectives(editor: vs.TextEditor, editBuilder: vs.TextEditorEdit) {
+        if (!editors.hasActiveDartEditor()) {
+            vs.window.showWarningMessage("No active Dart editor.");
+            return;
+        }
+
+        this.analyzer.editOrganizeDirectives({ file: editor.document.fileName }).then((response) => {
+            let edit: as.SourceFileEdit = response.edit;
+			if (edit.edits.length == 0) {
+                vs.window.showInformationMessage("No changes from organize directives.");
```

Comment:
Yeah, I was kinda torn, but since we don't do it for things like format, I think just saying out the way is best.

That said, I'm not against creating a little function that shows an info message for x seconds and using that, and then applying the same to reformat (and anywhere else we want to give feedback without a persistent message). What do you think?